examples/logging_configuration.md

```diff
@@ -0,0 +1,234 @@
+# Logging Configuration
+
+vLLM leverages Python's `logging.config.dictConfig` functionality to enable
+robust and flexible configuration of the various loggers used by vLLM.
+
+vLLM offers two environment variables that can be used, individually or in
+combination, to accommodate a range of logging configurations that vary in
+flexibility from:
+
+- vLLM's default logging configuration (least flexible)
+- coarse grained custom logging configuration
```

Comment:
Can you add a simple example or to a link to a relevant subsection here? for coarse grained & fined grained bullet points (the definition is not very straightforward)

Reply:
I tried adding some examples of how to use the configs to achieve these states and I think there's now an example for each state. Let me know if that's sufficient. I can definitely add a link related to each option if you think it'd be beneficial. I'm optimistic that the overall scheme is simpler now and it may not be needed.